docs/operating-scylla/rest.rst

```diff
@@ -2,7 +2,27 @@
 REST
 ----
 
-For each JMX operation, attribute get and set, Scylla exposes a matching REST API. You can interact with the REST API using :code:`curl` or using the Swagger UI available at :code:`your-ip:10000/ui`
+ScyllaDB exposes a REST API to retrieve administrative information from a node and execute 
+administrative operations. For example, it allows you to check or update configuration, 
+retrieve cluster-level information, and more.
 
 
+CLI for REST API
+^^^^^^^^^^^^^^^^^
+
+ScyllaDB ships with ``scylla-api-cli``, a lightweight tool that provides a command-line interface 
+for the ScyllaDB API.
+
+The ``scylla-api-cli`` tool allows you to:
+
+* List the API functions with their parameters and print detailed help for each function.
+* Invoke the API commands using the command line. 
+
+When invoking a function, ``scylla-api-cli`` performs basic validation of the function arguments and 
+prints the result to the standard output. You can use commonly available command line utilities
+to print the results in the JSON format.
+
+To avoid errors, you should prefer ``scylla-api-cli`` over cURL and similar HTTP tools for interacting 
```

Comment:
nit: s/cURL/curl/ as this is how the command is spelled on linux

Reply:
I  wanted to use a generic tool name rather than the actual command. Both versions of the tool name seem to be in use, but now I see that the curl documentation is actually using the name "curl", so I'll update it.